### PR TITLE
[DEVOPS-424] Simplify nix-based exchange workflow

### PR DESCRIPTION
**Pull request by jmitchell**

### CODE REVIEW

File: sample-wallet-config.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# If any customization is required, copy this file to
```
Comment (domenkozar) [inline sample-wallet-config.nix]: This file needs to be in `/scripts/launch/connect-to-cluster` for this code to work

File: default.nix [hunk 2]
```diff
@@ -1,5 +1,7 @@
 let
   localLib = import ./lib.nix;
+  walletConfigFile = ./custom-wallet-config.nix;
```
Comment (domenkozar) [inline default.nix]: (nitpick) I suggest we move this down, inside to `connect` function with a new `let .. in`

File: scripts/launch/connect-to-cluster/default.nix [hunk 3]
```diff
@@ -9,6 +9,8 @@
 , gitrev ? localLib.commitIdFromGitRepo ./../../../.git
 , walletListen ? "127.0.0.1:8090"
 , ekgListen ? "127.0.0.1:8000"
+, ghcRuntimeArgs ? "-N2 -qg -A1m -I0 -T"
+, additionalArgs ? ""
```
Comment (domenkozar) [inline scripts/launch/connect-to-cluster/default.nix]: (nitpick) additonalArgs -> additionalNodeArgs

**Comment by domenkozar** [comment]
Initially I wasn't happy with this approach, but on second thought it makes more sense to have declarative config over imperative args.

File: sample-wallet-config.nix (hunk 1 above)
Comment (jmitchell) [reply sample-wallet-config.nix]: If we were using JSON that would be true, but the path resolution works about when `custom-wallet-config.nix` is in the root of the repo and paths are specified relative to that directory.

Comment (domenkozar) [reply sample-wallet-config.nix]: Ah, you're right.

**Pull request merged**